### PR TITLE
Redirect Firebase issues to FirebaseExtended/flutterfire

### PR DESCRIPTION
**Pull request by kroikie**
## Description

Since FlutterFire has been moved to FirebaseExtended/flutterfire issues related to Firebase should be reported there. This PR creates an issue template that will alert developers that there is a better place to report Firebase issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/FlutterFire.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: FlutterFire
+about: FlutterFire has been moved to [FirebaseExtended/flutterfire](https://github.com/FirebaseExtended/flutterfire).
+       All issues related to FlutterFire should be filed and will be
```
Comment (collinjackson) [inline .github/ISSUE_TEMPLATE/FlutterFire.md]:
```suggestion
       All issues related to FlutterFire should be filed there.
```

File: .github/ISSUE_TEMPLATE/FlutterFire.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+       addressed there.
```
Comment (collinjackson) [inline .github/ISSUE_TEMPLATE/FlutterFire.md]:
```suggestion
```

**Review by collinjackson — APPROVED** [review]

**Comment by goderbauer** [comment]
Is this ready to be submitted?

**Comment by goderbauer** [comment]
I am going to close this since there was no response.

**Pull request closed without merging**